src/mtx/CommandBinaryBuffer.ts

```diff
@@ -351,6 +357,20 @@ class CommandBinaryBuffer extends BinaryBuffer {
         super(dataOrLength, false);
     }
 
+    getFrameHeader (): IFrameHeader {
+        return {
+            type: this.getUint8(),
+            destination: this.getUint16(),
+            source: this.getUint16()
+        };
+    }
+
+    setFrameHeader ( header: IFrameHeader ) {
```

Comment:
```suggestion
    setFrameHeader ( frameHeader: IFrameHeader ) {
```

for better readability

Reply:
fixed